SauceDemo/Test/CheckoutTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using SauceDemo.PageObject;
+
+namespace SauceDemo.Test
+{
+    internal class CheckoutTest : BaseTest
+    {
+        [Test]
+        public void Checkout_BuyBackpack_StandartUser()
+        {
+            var user = new UserModel()
+            {
+                FirstName = "John",
+                LastName = "Doe",
+                PostalCode = "ABC123"
+            };
+
+            var inventoryPage = new LoginPage(driver)
+                .OpenPage()
+                .LoginAsStandartUser()
+                .AddBackpack()
+                .GoToCartPage()
+                .GoToCheckoutPage();
+
+            inventoryPage.FillUserData(user);
+
+            var checkoutPage = new CheckoutPage(driver);
```

Comment:
and after that You  dont need this new object

Reply:
Changed here https://github.com/khzrx/TeachMeSkills/pull/12